packages/preview/modern-cv/0.2.0/template/resume.typ

```diff
@@ -0,0 +1,95 @@
+#import "@local/modern-cv:0.2.0": *
+
+#show: resume.with(
+  author: (
+      firstname: "John", 
+      lastname: "Smith",
+      email: "js@example.com", 
+      phone: "(+1) 111-111-1111",
+      github: "DeveloperPaul123",
+      linkedin: "Example",
+      address: "111 Example St. Example City, EX 11111",
+      positions: (
+        "Software Engineer",
+        "Software Architect",
+        "Developer"
+      )
```

Comment:
There are some double indents here in various places, which might you want to remove.